Training_25/Program.cs

```diff
@@ -3,11 +3,37 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Assignment 4: Chess Board
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
 namespace Training_25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      OutputEncoding = Encoding.UTF8;
+      string[][] board =
+      [
+        ["♜","♞","♝","♛","♚","♝","♞","♜"],
+        ["♟","♟","♟","♟","♟","♟","♟","♟"],
+         ..Enumerable.Range(0, 4).Select(_ => EmptyRow()),
+        ["♙", "♙", "♙", "♙", "♙", "♙", "♙", "♙"],
+        ["♖", "♘", "♗", "♕", "♔", "♗", "♘", "♖"]
+        ];
+      PrintBoard (board);
+      WriteLine ("Press any key to exit..."); ReadKey ();
    }
-}
+
+   static string[] EmptyRow () => [.. Enumerable.Repeat (" ", 8)];
+
+   static void PrintBoard (string[][] board) {
+      string top = "┏━━━┳━━━┳━━━┳━━━┳━━━┳━━━┳━━━┳━━━┓";
+      string divider = "┣━━━╋━━━╋━━━╋━━━╋━━━╋━━━╋━━━╋━━━┫";
+      string bottom = "┗━━━┻━━━┻━━━┻━━━┻━━━┻━━━┻━━━┻━━━┛";
+      WriteLine (top);
+      for (int i = 0; i < 8; i++) {
+         WriteLine ($"┃{string.Join ("┃", board[i].Select (x => $" {x} "))}┃");

```

Comment:
This can be simplified as:
```suggestion
         WriteLine ($"┃ {string.Join (" ┃ ", board[i]))} ┃");
```